Ports/ruby/version.sh

```diff
@@ -1,6 +1,6 @@
 RUBY_MAJOR="3.0"
-RUBY_TEENY="2"
+RUBY_TEENY="4"
```

Comment:
Was this included by accident?

Reply:
Not really, I figured updating the port and removing one of the patches is easier than rebuilding ruby, updating the patches, then updating ruby and removing one of the patches.